storybook/pages/SwapInputPanelPage.qml

```diff
@@ -78,7 +78,11 @@ SplitView {
 
                 currencyStore: d.adaptor.currencyStore
                 flatNetworksModel: d.adaptor.swapStore.flatNetworks
-                processedAssetsModel: d.adaptor.processedAssetsModel
+                processedAssetsModel: d.adaptor.walletAssetsStore.groupedAccountAssetsModel
+
+                selectedNetworkChainId: d.swapInputParamsForm.selectedNetworkChainId
+                selectedAccountAddress: d.swapInputParamsForm.selectedAccountAddress
+                nonInteractiveTokensKey: receivePanel.selectedHoldingId
```

Comment:
we can use the input form itself here instead?

Reply:
it's a visual property :shrug:

Reply:
yes but for consistency maybe e can use d.swapInputParamsForm.toTokensKey?